R/oc_points.R

```diff
@@ -0,0 +1,111 @@
+#' List of points for opencage queries
+#'
+#' Create a list of points (latitude/longitude coordinate pairs) for opencage
+#' queries.
+#'
```

Comment:
Do you want to refer to the R package or the service? If the service, then should probably be OpenCage instead of opencage.

Reply:
I guess I meant to refer to the package, but the service makes more sense, indeed.